sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -39,6 +39,40 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
 
   setupTestData()
 
+  test("describe table column") {
```

Comment:
How about creating `describe-table-column.sql` and move them there? It also can help reviewers read the output.